rust_src/remacs-lib/docfile.rs

```diff
@@ -130,6 +129,14 @@ pub extern "C" fn scan_rust_file(
                     "F", attr_props.name, docstring, docstring_usage
                 );
             }
+        } else if line.starts_with("def_lisp_sym!(") {
+            lazy_static! {
+                static ref RE: Regex = Regex::new(r#"def_lisp_sym!\((.+),\s+"(.+)"\);"#).unwrap();
```

Comment:
This could capture too much if there's a sequence of def_lisp_sym. I suggest you make this regexp non-greedy, using `+?` instead of `+`.

Reply:
Fair catch. Update on the way.